config/install/field.field.node.alert_banner.field_alert_start_date.yml

```diff
@@ -0,0 +1,21 @@
+uuid: 7d2337c3-e24c-40f4-9361-8252f35f1ef1
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_alert_start_date
+    - node.type.alert_banner
+  module:
+    - datetime
+id: node.alert_banner.field_alert_start_date
+field_name: field_alert_start_date
+entity_type: node
+bundle: alert_banner
+label: 'Start date'
+description: ''
```

Comment:
Consider adding a description to help content editors understand the field's purpose:
```yaml
description: 'Alert will only display on or after this date and time. Leave empty to display immediately.'
```